std_msgs/Header                      header
aist_aruco_ros/PointCorrespondence[] point_correspondences
